### PR TITLE
feat: added nav component

### PR DESCRIPTION
**Pull request by aryan-13**
- Added the nav component
Preview:
image

### CODE REVIEW

**Review by anshikaa12 — APPROVED** [review]
looks good to merge

File: components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+/* HOVER EFFECT ON NAV ITEMS */
+.menu-item:hover,
+.nav-child:hover {
+    color: var(--color-tertiary-dark);
+    transform: scale(1.1);
+}
+
+@media (max-width: 1120px) {
```
Comment (amarphule) [inline components/navigation.css:66]: you can use `rem` instead `px`

**Review by amarphule — COMMENTED** [review]
Great work @aryan-13 code looks good and well-formatted.

File: components/navigation.css (hunk 1 above)
Comment (aryan-13) [reply components/navigation.css:66]: I think in media queries, px are used!

**Pull request merged**